LibreNMS/Alert/Transport/Api.php

```diff
@@ -52,10 +53,23 @@ private function contactAPI($obj, $api, $method)
     {
         $method = strtolower($method);
         list($host, $api) = explode("?", $api, 2);
+        $api = trim($api);
+
         foreach ($obj as $k => $v) {
-            $api = str_replace("%" . $k, $method == "get" ? urlencode($v) : $v, $api);
+            $api = str_replace("{{ $" . $k . " }}", $method == "get" ? urlencode($v) : $v, $api);
```

Comment:
This is an adaptation of the old str_replace but using new syntax for blades
- looks similar to the other Blade based templates but is not at all one. 
- allow proper urlencode to take place for GET

Reply:
Blades are now in use with the new config options.